.github/workflows/on-dev-merge.yml

```diff
@@ -29,4 +29,4 @@ jobs:
       if: ${{ steps.find.outputs.issue }}
       with:
         issue: ${{ steps.find.outputs.issue }}
-        transition: "Move to 'Deployed in Integration'"
+        transition: "Move to 'Deployed on Integration'"
```

Comment:
Je ne suis pas sûr que cela corrige le problème.
Quand je lis le message d'erreur
> { id: 691, name: Move to 'Deployed in Staging' } transitions issue to 'Deployed on Integration' status.

je comprend que la transition `Move to 'Deployed in Staging'` déplace le ticket dans l'état `'Deployed on Integration'`.
Ce serait peut-être plus simple et plus robuste d'utiliser l'id de la transiton (`691`).

Reply:
J'ai l'impression que je me suis perdu dans les runs de github actions, j'avais ça dans un onglet: https://github.com/1024pix/pix/runs/1516657547

Des fois c'est la transition `701` et des fois `691` :exploding_head:

Reply:
Etonnant, cela a fonctionné sur pour ce ticket : https://github.com/1024pix/pix/runs/1517672110

Reply:
Je confirme, ça a fonctionné à nouveau pour https://github.com/1024pix/pix/pull/2240

Reply:
J'attend quelques merges pour voir si ce maraboutage continue.

Reply:
Après investigation, il y a une différence de workflow entre les teams et la partie cross-team qui est resté en "staging".

Reply:
Tout s'explique !